src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -218,6 +218,7 @@ export function CurrencySearch({
                 showImportView={showImportView}
                 setImportToken={setImportToken}
                 showCurrencyAmount={showCurrencyAmount}
+                isLoading={balancesIsLoading}
```

Comment:
Have this be `balancesIsLoading && !timeoutElapsed` or something similar so that the loading bars go away if it takes too long to load.

Reply:
sounds good, set to 2 seconds